pom.xml

```diff
@@ -14,7 +14,7 @@
   <packaging>hpi</packaging>
 
   <properties>
-    <changelist>999999-SNAPSHOT</changelist>
```

Comment:
Why also change the changelist here?
Based on the build before this change it was sufficient just to change `-Dchangelist=`
https://github.com/seansaleh/artifact-manager-s3-plugin/suites/6207879953/artifacts/219143563

Reply:
For some reason I thought it didn't work and I needed to remove this. But I must've made a mistake checking the `hpi` file from a different build because the change was suficient. I'll revert this change.